docs/builders/email_landing_page.rst

```diff
@@ -3,40 +3,46 @@
 Email & Landing Page Builder
 ############################
 
-.. vale on
 
 Since :xref:`Mautic 3`, Mautic has shipped with an updated, modern builder for creating Emails and landing pages. 
 In :xref:`Mautic 4` it's the default builder.
 
+.. vale on
```

Comment:
Why have you moved the Vale On command to here? It should remain where it was, and you should be fixing the Vale error flagged.